### PR TITLE
Google Cloud deployment

### PR DESCRIPTION
**Pull request by 4LineTime**
settings.py is where the fun stuff is, and it is very minimal. As I'm sure you know, most of the action is in the command line and the google cloud web interface. 
Our url is here:
https://clear-booking-309320.uc.r.appspot.com/

Please break it if you can.

This branch also happens to add a ton of files in the static folder. I'm not sure if that is where they are supposed to be. I followed Clara's slides and there they are. These are direct copies of files elsewhere.

### CODE REVIEW

**Comment by claraj** [comment]
> 
> This branch also happens to add a ton of files in the static folder. I'm not sure if that is where they are supposed to be. I followed Clara's slides and there they are. These are direct copies of files elsewhere.

That's completely fine. You work on the originals, and then collectstatic gathers them all up and copies into the static folder, and those files are used for the live site.

**Comment by johnnyvuanly** [comment]
I see that the app is deployed successfully at GCP! Great work Orion. I'm guessing all of these files added were from GCP?

File: .gitignore [hunk 1]
```diff
@@ -11,10 +11,15 @@ __pycache__
...
+cloud_sql_proxy_x64.exe
 
 #Google Cloud Files
 app.yaml
 .gcloudignore
 cloud_sql_proxy_x64.exe
 .env
 image-upload-credentials.json
+
+requirements.txt
```
Comment (claraj) [inline .gitignore:25]: I don't think you should ignore this. both github and GCP need this file

File: .gitignore [hunk 2]
```diff
@@ -11,10 +11,15 @@ __pycache__
...
 venv
 env
+.gcloudignore
+app.yaml
+cloud_sql_proxy_x64.exe
 
 #Google Cloud Files
 app.yaml
 .gcloudignore
 cloud_sql_proxy_x64.exe
```
Comment (claraj) [inline .gitignore:21]: There's some duplicates here, remove extra cloud_sql_proxy_x64.exe

**Review by claraj — CHANGES_REQUESTED** [review]

**Comment by ajgoranson** [comment]
Tested and used all the application features, it is working great!

**Pull request merged**